src/main/java/org/fundacionjala/at13/spaceinvaders/GameOver.java

```diff
@@ -4,19 +4,20 @@
 import java.awt.*;
 import java.awt.event.*;
 import javax.swing.ImageIcon;
+import java.util.Timer;
 
 public class GameOver {
 
```

Comment:
good job jessi maybi this line maybe this line could be removed

Reply:
Thank you. I just deleted that line